reproc++/include/reproc++/reproc.hpp

```diff
@@ -203,6 +203,10 @@ class process {
   REPROCXX_EXPORT std::pair<int, std::error_code>
   stop(stop_actions stop) noexcept;
 
+  /*! `reproc_pid` */
```

Comment:
I only comment stuff in reproc++ if it does something extra or different than the C function. This just delegates to reproc so the comment isn't necessary.